src/collections/LinkedList.java

```
@@ -0,0 +1,102 @@
+package collections;
+
+public class LinkedList<T> implements List<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public T get(int index) {
```

Also, you could make optimization. If index<size/2 start your search from `head`, else from `tail`.
And what if index<0 or index>=size. It is better to throw an exception. Create your own class exception (better unchecked) and use it.

---

Upper comment is not resolved